distributions/nuxt/package.json

```diff
@@ -39,9 +39,12 @@
     }
   ],
   "files": [
-    "dist"
+    "dist",
+    "types/index.d.ts",
```

Comment:
This typings are directly related to `@nuxt/vue-app` package. If we can't reference typings from a dependency package into main `index.d.ts` (It seems it should be published under `@types`), I suggest at least renaming it into `vue-app.d.ts` and importing in `index.d.ts` :D 	@kevinmarrec

Reply:
(See my comment in the PR before reading this)

Best thing seems to move the types under `@nuxt/vue-app`, and people will enable these typings by refering `types: ['@nuxt/vue-app']` in their `tsconfig.json`.